src/app.js

```diff
@@ -65,13 +58,18 @@ app.get(oauth2CallbackPath, async (req, res) => {
     });
 
     req.session.userId = dbUser.id;
+
     logger.info(`role: ${req.session.userId} ${req.session.referrerPath}`);
     res.redirect(join(process.env.TTA_SMART_HUB_URI, req.session.referrerPath));
   } catch (error) {
     logger.error(`Error logging in: ${error}`);
     res.status(INTERNAL_SERVER_ERROR).end();
   }
 });
+app.use((req, res, next) => {
+  req.session.nowInMinutes = Math.floor(Date.now() / 60e3);
+  next();
+});
```

Comment:
Is this required if there is no max-age or expiration set?

Reply:
Good catch! This was only for initial tests and was slated to be removed.